assets/scss/dropdown.scss

```diff
@@ -31,10 +31,12 @@
     text-decoration: none;
   }
 
-  &.more-facets {    
```

Comment:
Changes to this stylesheet are to reflect changes to the more filters dropdown component to make better semantic use of HTML form elements.